react/cra-template/template.json

```diff
@@ -63,6 +64,7 @@
       "typescript-fsa-reducers": "^1.2.1"
     },
     "scripts": {
+      "prestart": "chmod +x ./env.sh && dotenv -e .env.development.local -e .env.development -e .env.local -e .env ./env.sh ./public/env.js",
```

Comment:
run env.sh before `yarn start` to ensure a env.js file exists in the public directory with the contents of any of the .env files that may exist locally.  This follows the same overlay rules as defined by https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used